### PR TITLE
Fix README to pass markdownlint checks

### PR DESCRIPTION
**Pull request by ruzickap**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # cna-installer
+
 ```bash
 cna-installer is a binary that installs, sets-up and configures a
-kubernetes cluster with the CNA stack applications.
+Kubernetes cluster with the CNA stack applications.
```
Comment (trawler) [inline README.md:5]: I think this is a false-positive, since it's attempting to capitalise a word in mid-sentence.

Comment (ruzickap) [reply README.md:5]: Not sure - I believe you should always write "Kubernetes cluster" with capital "K". For example: https://kubernetes.io/docs/setup/

Comment (trawler) [reply README.md:5]: got it :) lgtm, then.
you might want to fix the command output then in https://github.com/trawler/cna-installer/blob/ff741bdb2b4b6e33474fd213c019ee1486bc0473/cmd/main.go#L29 ;)

Comment (ruzickap) [reply README.md:5]: Done...

**Comment by trawler** [comment]
@ruzickap Please rebase against master and merge.

**Pull request merged**